src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -1,7 +1,7 @@
+import ReactJson from '@microlink/react-json-view';
 import _ from 'lodash/fp';
 import { Fragment, useRef, useState } from 'react';
 import { div, h } from 'react-hyperscript-helpers';
-import ReactJson from 'react-json-view';
```

Comment:
Before and after:

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/5aa66f14-e842-43fb-b590-667ec7a68f0e)

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/87adf9f8-20c2-4935-b4e6-d1a92d4d677a)